module2/lessons/js_intro_to_dom_manipulation.md

```diff
@@ -0,0 +1,393 @@
+
+Include a helpful string in each console log to serve as a 'label' for what you're printing. Separate the string from the variable with a comma.
+```js
+console.log('title', title)
+```
+</section>
+
+## Updating Content of an Element
+
+Now that we know how to query elements, let's look at how to update the content of an element we've accessed.
+
+```javascript
+var title = document.querySelector('h1');
+```
+
+When we query a DOM element and store it in a variable, we now have access to all the properties and methods that are attached to that DOM element. We won't learn about all of them today. Just a few to get you started.
+
+### `innerText`
+
+The [innerText](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText) property holds the text that lives inside of a given element. In our codepen, the text content inside of the `h1` is `Elephants!`.  We can access that text content by accessing the element's `innerText` property. We can use the `=` assignment operator to change it.
+
+<section class="call-to-action">
+### Try It Together 
+
+In our codepen let's:
+- Change our console.log()s so that they print each element's innerText
+  - What happens when we try to print the `.innerText` of our `<img>` element?
+ 
+- Write a line of code that changes the title's text to a different animal
+- Write a line of code that changes the elephant fact to a different animal fact
+
+**Consider:**  
+How would we change the elephant image to a different image?  Hmm...   
+We've seen from the console log that we won't be able to use `innerText` property for the image.  
+Looking at the `<img>` in the html, is there a different property we might be able to manipulate?
+</section>
+
+
+<section class="call-to-action">
+### Your Turn
+
+Fork [this Report Card CodePen](https://codepen.io/turingschool/pen/rNEqEVw).  In the `JS` file of the CodePen, complete the steps listed below.  Work through each bullet one at a time.
+
+- Declare a variable `studentName` and assign the value of the element with the class of `student` to it
+- Declare a variable `bestGrade` and assign the value of the element with a class of `grade-2` to it
+
+Now that we've queried the elements we need, we're going to change the data being displayed to users.  So we need somewhere to store the new data that we're going to display instead.
+- Declare a variable `newData` to store an object with keys of `studentName` and `bestGrade`.  Make the value of each key be the new name, school, and grade that you'll want to change the display to.   
+
+Now, change the text of each queried element to display the data you saved in your newData object.
+- Change the `innerText` of the `studentName` variable to be the `studentName` value from your `newData` object (use dot notation)
+- Change the `innerText` of the `bestGrade` variable to be the `bestGrade` value from your `newData` object
+</section>
+
+## Event Listeners
```

Comment:
I think it might be worth adding a block of code here to show an example of an event listener. I know we'll show them live, but we want this lesson page to be a resource they can revisit later. 

just something small like:

Here's the basic skeleton of an event listener:
```
element.addEventListener('click', doSomething);
```